src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -198,28 +217,39 @@ protected int doWork() {
         }
 
         // Construct and write the outputs
-        final Histogram<Integer> histo = new Histogram<Integer>("coverage", "count");
+        final Histogram<Integer> depthHistogram = new Histogram<Integer>("coverage", "count");
         for (int i = 0; i < HistogramArray.length; ++i) {
-            histo.increment(i, HistogramArray[i]);
+            depthHistogram.increment(i, HistogramArray[i]);
         }
 
         // Construct and write the outputs
-        final Histogram<Integer> baseQHisto = new Histogram<Integer>("value", "baseq_count");
+        final Histogram<Integer> baseQHistogram = new Histogram<Integer>("value", "baseq_count");
         for (int i=0; i<baseQHistogramArray.length; ++i) {
-            baseQHisto.increment(i, baseQHistogramArray[i]);
+            baseQHistogram.increment(i, baseQHistogramArray[i]);
         }
 
-        final WgsMetrics metrics = generateWgsMetrics();
-        metrics.GENOME_TERRITORY = (long) histo.getSumOfValues();
-        metrics.MEAN_COVERAGE = histo.getMean();
-        metrics.SD_COVERAGE = histo.getStandardDeviation();
-        metrics.MEDIAN_COVERAGE = histo.getMedian();
-        metrics.MAD_COVERAGE = histo.getMedianAbsoluteDeviation();
+        // Construct and write the outputs
+        final Histogram<Integer> baseQHetHistogram = new Histogram<Integer>("value", "baseq_count");
+        final int BASEQ_MAX = 50;
+        final Integer[] x = new Integer[50];
+        IntStream.range(0, BASEQ_MAX).forEach(i -> x[i] = i);
+        baseQHetHistogram.prefillBins(x);
+
+        for (int i=0; i<baseQHetSensHistogram.length; ++i) {
+            baseQHetHistogram.increment( i < 17 ? 0 : i, baseQHetSensHistogram[i]);
```

Comment:
Yes I will, good call, thank you. It's the base quality minimum cut off for haplotype caller. Theoretical sensitivity works best with more quality information than the original base quality histogram had in collectwgs (original cuts off at 20 and throws out everything below that)